renderers/bases.py

```diff
@@ -32,21 +32,31 @@ def render(self):
 
   def __render_base_outline(self, base):
     # Hollow diamonds are a popular homework problem but IDGAF
-    self.canvas.SetPixel(base['x'] - 3, base['y'], *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] - 2, base['y'] - 1, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] - 2, base['y'] + 1, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] - 1, base['y'] - 2, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] - 1, base['y'] + 2, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'], base['y'] - 3, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'], base['y'] + 3, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] + 1, base['y'] - 2, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] + 1, base['y'] + 2, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] + 2, base['y'] - 1, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] + 2, base['y'] + 1, *ledcolors.scoreboard.text)
-    self.canvas.SetPixel(base['x'] + 3, base['y'], *ledcolors.scoreboard.text)
+

```

Comment:
Move the "size" of the bases to the json file and create the bases with a loop.

Reply:
My math isn't all that great there either, the box that actually fills that larger diamond for a runner on base leaves an empty pixel on all the topmost points in the diamond because i just increased the offset for the runner by 1.